src/MenuItem.js

```diff
@@ -35,6 +37,8 @@ export default class MenuItem extends React.Component {
       );
     }
 
+    const {className, style, href, onClick, ...props} = this.props;
```

Comment:
Not quite - the destructuring default only works on `undefined`, where as the `href || ''` replaces any falsy values. I don't know if that part matters, but it'd be an additional behavior change I didn't want to introduce.